added sticky to side of page...thoughts?

Tom-Tee opened the pull request:
Hello guys,

I thought it'd be good to have a sticky sidebar to the bike show page...Nothing crazy at the moment, minimal styling. 

Could do a heap of stuff with it. Thoughts?

https://user-images.githubusercontent.com/68646479/106975478-bd1fa100-67aa-11eb-8722-905a8e9d5a6e.mov

Tom-Tee committed:
added sticky to side of page...thoughts?

julianharr commented:
@Tom-Tee I like the idea! It looks like it's in the way right now though. Maybe we can fix the nav bar to the top of the page then create a sticky sidebar when you scroll down like you have it, kind of like this Codepen:
https://codepen.io/reyschelle/pen/WNvQLGp

Tom-Tee committed:
added some responsiveness, edited the sticky card (still not sure about it) and did some slight styling

Tom-Tee commented:
Did a little more work on this tonight...

- Added some responsiveness to the page
- Made the if/else statement work (if bike is available)
- Added some borders/shadows
- Worked on the sticky card (still not sure about this 🧐  )

Still not 100% but definitely an improvement (kind of feel it's missing something..)

Cheers boys,

https://user-images.githubusercontent.com/68646479/107035432-474c2180-680c-11eb-9766-74b0aa715929.mov

julianharr commented:
Looks fresh!!
